pkg/database/mysql/deploy.go

```diff
@@ -485,6 +485,11 @@ func buildRemoveIndexStatements(m *MysqlConnection, tableName string, mysqlTable
 	for _, currentIndex := range currentIndexes {
 		isMatch := false
 		for _, desiredIndex := range mysqlTableSchema.Indexes {
+			// if there's no name on the desired index,
+			// generate one
```

Comment:
Would we want to generate a state file like terraform does so we could more easily determine the desired state/drift and for storing programmatically generated values for resources?

Reply:
Interesting. I don't think we've had a discussion about this. for simplicity, hopefully we can continue to just let the database schema be the state, and always calculate it. A large value of schemahero is that we can run it when we don't know the "state" of the database and calculate it at runtime.

Reply:
Sounds good, thanks for sharing the insight. =)